examples/arkane/species/Toulene_Hindered_Rotor_SemiClassicalND/toluene_HinderedRotor.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python

```

Comment:
I don't understand the example - toluene has just one rotor (as this file correctly states), so why is this an ND example? Is there a reason to use ND for N=1?

Reply:
It's a good way to test the ND code against the regular HinderedRotor.  In theory it should be more accurate if quantum is off and it isn't given the rotor scan in one file because it accounts for changes in reduced moment of inertia during the scan.  Right now the Q2DTor example isn't ideal for testing it because the Q2DTor files don't put the minimum at (0,0) in the PES, which is problematic for HinderedRotorClassicalND.

Reply:
So maybe add it as a test? It might confuse users as an example

Reply:
It's the only fully functioning example I have available of how to call HinderedRotorClassicalND, which is useful for a user.  (I've run 2D with the code, but it was using the CH2CHOOH example in a hacky way that doesn't work in Arkane).  There are reasons to use ND for N=1, it's about as accurate as HinderedRotor is (it's semiclassical should be more accurate than HinderedRotor semiclassical since the ND accounts for changes in internal reduced moment of inertia).